### PR TITLE
stdout log instructions enhancement

### LINKED ISSUE #5891: Enhance content for stdout log writing
Feedback from @Scott_Moses ...

> I  had the cryptic error on asp.net core "An error occurred while starting the application." and could not get the log file to output, despite having created the log directory and set the webconfig per example.  Buried in the project config documentation, was a note to give write permissions to the apppool for the log directory.  Would help if you included this info above.  Would not hurt to list the example of IIS Apppool\MyAppPool to add the pool to the directory permissions.

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #5891 

* Add a bit more info to the IIS config topic
  - The path must exist
  - The app pool must have permission to write to the location
* Add a bit of content with an explicit example of the user identity to the linked ANCM ref topic (and re-organize a bit)

Thanks to @Scott_Moses for the heads-up.

### CODE REVIEW

**Comment by guardrex** [comment]
@Rick-Anderson @scottaddie I doubt that we need an engineering review on this one. It's just adding some well-known content with a few minor organizational changes.

File: aspnetcore/host-and-deploy/aspnet-core-module.md [hunk 1]
```diff
@@ -108,9 +108,13 @@ If the ASP.NET Core Module fails to launch the backend process or the backend pr
...
 
-The log file name is composed by appending the timestamp, process ID, and file extension (*.log*) to the last segment of the `stdoutLogFile` path (typically *stdout*) delimited by underscores. If the `stdoutLogFile` path ends with *stdout*, a log for an app with a PID of 1934 created on 2/5/2018 at 19:42:32 has the file name *stdout_20180205194132_1934.log*.
+Logs aren't rotated, unless process recycling/restart occurs. It's the responsibility of the hoster to limit the disk space the logs consume.
+
+Using the `stdout` log is only recommended for troubleshooting app startup issues. Don't use the stdout log for general app logging purposes. For routine logging in an ASP.NET Core app, use a logging library that limits log file size and rotates logs. For more information, see [third-party logging providers](xref:fundamentals/logging/index#third-party-logging-providers).
```
Comment (scottaddie) [inline aspnetcore/host-and-deploy/aspnet-core-module.md]: Don't use the stdout --> Don't use the `stdout`

File: aspnetcore/host-and-deploy/iis/index.md [hunk 2]
```diff
@@ -191,7 +191,7 @@ When deploying apps to servers with [Web Deploy](/iis/publish/using-web-deploy/i
...
 
-   **Important!** Only use the stdout log to troubleshoot app startup failures. Never use stdout logging for routine app logging. There's no limit on log file size or the number of log files created. For more information on the stdout log, see [Log creation and redirection](xref:host-and-deploy/aspnet-core-module#log-creation-and-redirection). For information on logging in an ASP.NET Core app, see the [Logging](xref:fundamentals/logging/index) topic.
+   **Important!** Only use the stdout log to troubleshoot app startup failures. Never use stdout logging for routine app logging. There's no limit on log file size or the number of log files created. The app pool must have write access to the location where the logs are written. All of the folders on the path to the log location must exist. For more information on the stdout log, see [Log creation and redirection](xref:host-and-deploy/aspnet-core-module#log-creation-and-redirection). For information on logging in an ASP.NET Core app, see the [Logging](xref:fundamentals/logging/index) topic.
```
Comment (scottaddie) [inline aspnetcore/host-and-deploy/iis/index.md]: - Convert this to the IMPORTANT widget
- Should any of these "stdout" references by code-fenced for consistency?

Comment (guardrex) [reply aspnetcore/host-and-deploy/iis/index.md]: It wasn't my impression that I've been code fencing them, but these topics go back quite a way in repo history. I'm not surprised to see a bit of this. I'll add a commit to this PR to fix them all everywhere just so we can knock it out here. However, I prefer them *not* to be code fenced. imo code-fencing slows down reading comprehension and should only be used when it's necessary to separate code (and links) from the flow of the text.

@Rick-Anderson Thoughts?

Comment (guardrex) [reply aspnetcore/host-and-deploy/iis/index.md]: Yes ... as I suspected ... there are only six instances where "stdout" is code-fenced across the entire doc set. There are 30 instances where it isn't code fenced. I prefer them *not* code fenced, so I'd like to remove the code fencing from these six instances.

Comment (scottaddie) [reply aspnetcore/host-and-deploy/iis/index.md]: Not code-fenced is fine with me. Lack of consistency is the bigger issue to address.

**Comment by guardrex** [comment]
@scottaddie That build just :boom: for no obvious reason.

I'll try to close and re-open, but I have a 😨 *bad feel'in* 😨 about this.

**Comment by guardrex** [comment]
I love the way the email says that it **succeeded** ........ **_to FAIL!_** :grinning: lol

I can use that in my personal life, too. :smile:

capture

**Comment by scottaddie** [comment]
@guardrex I ran into this problem this morning with a PR I created.

**Pull request merged**